backend/tests/v5/unit/models/users.test.js

```diff
@@ -49,6 +50,7 @@ const testGetAccessibleTeamspaces = () => {
 				roles: [
 					{ db: 'ts1', role: 'a' },
 					{ db: 'ts2', role: 'b' },
+					{ db: 'admin', role: 'user' },
```

Comment:
use generateRandomString if you don't care about the role name, if we do then we should probably be using the constant from `src`